### DISCUSSION TITLE
web-pag替换composition有内存泄漏问题

### DISCUSSION THREAD

**Discussion by wwenying** [Tencent/libpag#2215] [Q&A] [answered]
pag主要使用场景是：定时load pag-file -->  pagcomposition.addLayer --> setComposition；
已经在每次load file后调用pagfile.destroy，和每次替换完composition后调用composition.destroy，但是还是有内存泄漏问题；
在这种场景下：定时加载新file并替换composition的场景，应该怎么解决内存泄漏呢？
<img width="840" alt="企业微信截图_5b0cfc54-11cb-4629-a423-2990006aa2d6" src="https://github.com/Tencent/libpag/assets/49852453/116018e5-cce8-4929-a902-f375feb7903e">
<img width="831" alt="企业微信截图_8156ddcd-aff3-44f5-9236-563436e8976a" src="https://github.com/Tencent/libpag/assets/49852453/dd8d8f95-f115-4e5d-901b-321c3d9be6d9">

**Comment by zhalice2011** [comment, marked as answer]
`pagView.freeCache` 释放缓存

<img width="825" alt="image" src="https://github.com/Tencent/libpag/assets/24289349/0fea5e7d-df3f-4b73-bdb2-28315c6e3f5e">

<img width="502" alt="image" src="https://github.com/Tencent/libpag/assets/24289349/89877d30-7be8-4506-b30d-8fcc31bc8a62">
